app/assets/javascripts/govuk_frontend_tab_patch_for_using_browser_back.js

```diff
@@ -0,0 +1,12 @@
+window.onpopstate = function(event) {
+  if (event.state == null) return
+  if (event.state.name == 'CustomHistoryManipulator') {
+    window.location = document.location;
+  }
+};
+
+$(document).on('click', '.govuk-link', function(e) {
```

Comment:
This should be flexible enough so that any links added and remove to the school#show page don't have to remember to add a custom class for this to listen to. This seems okay since I'm only including that JS on this page.